src/services/HttpService.ts

```diff
@@ -0,0 +1,160 @@
+import axios from './axios';
+
+/**
+ * Bare bones http service over an underlying implementation.
+ */
+
+/**
+ * Url replacements
+ */
+export type Replacements = Record<string, string>;
```

Comment:
TIL .. we have been using types like `{ [key: string]: string}` when we can just use `Record<string, string>` ,, handy

Reply:
useful doc https://www.typescriptlang.org/docs/handbook/utility-types.html

Reply:
Nice!

Reply:
I think we have some Record<> in some parts of the code